net-p2p/resilio-sync/files/resilio-sync.initd

```diff
@@ -1,5 +1,5 @@
 #!/sbin/openrc-run
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 command="/usr/bin/rslsync"
```

Comment:
can't comment on exact lines, but
Please don't use over generic variable names like `USER` and `GROUP`.
try something unique, like `RSLSYNC_USER`

imagine what will happen if user deletes conf.d file or comments out default user line. not sure if openrc guards against such scenario, but `USER` variable could be inherited from environment and daemon will be launched as root.

Reply:
it'll be even better if in addition to unique variables you'll use default commented-out confd file.

Reply:
I think that we are talking about — command_user="${USER:-rslsync}:${GROUP:-rslsync}".

> imagine what will happen if user deletes conf.d file or comments out default user line.

If a user comments out USER/GROUP nothing will be changed.

> it'll be even better if in addition to unique variables you'll use default commented-out confd file.

That is exactly what I am doing. If a user removes conf.d it will lose logs, that's all.
Moreover, resilio-sync-user.confd is commented out by default.

Reply:
@SpiderX what @gyakovlev means is that `${USER}` is a special bash variable expanding to the current user.

Reply:
yes, what @mgorny said.
`${USER}` holds current username in bash and very likely is present in the environment.

in case with `files/resilio-sync.confd` it comes with uncommented `USER/GROUP` definitions. 
if user deletes that file (`/etc/conf/d/resilio-sync`) or comments out the variables, there is a possibility that value of `USER` will be inherited from environment at the time of launch.

```sh
ya@reaper ~> echo '#!/bin/bash' > test.sh
ya@reaper ~> echo 'echo ${USER}' >> test.sh
ya@reaper ~> chmod +x test.sh 
ya@reaper ~> ./test.sh 
ya
ya@reaper ~> bash test.sh 
ya
```

Reply:
Variables are prefixed with RSLSYNC_ now.
Thanks.